src/server/json_family.cc

```diff
@@ -138,40 +128,47 @@ error_code JsonReplace(json& instance, string_view& path, JsonReplaceCb callback
   return ec;
 }
 
-bool JsonErrorHandler(json_errc ec, const ser_context&) {
-  VLOG(1) << "Error while decode JSON: " << make_error_code(ec).message();
-  return false;
-}
-
-optional<json> ConstructJsonFromString(string_view val) {
-  error_code ec;
-  json_decoder<json> decoder;
-  basic_json_parser<char> parser(basic_json_decode_options<char>{}, &JsonErrorHandler);
+OpStatus UpdateEntry(const OpArgs& op_args, std::string_view key, std::string_view& path,
+                     JsonReplaceCb callback, JsonReplaceVerify verify_op = JsonReplaceVerifyNoOp) {
```

Comment:
Why is the path passed as a mutable string view... It don't see where its changed

Reply:
I coped the arg list from JsonReplace, didn't notice this ref, I will remove this from both places, we're not changing this, so its not making any sense.